.eslintrc

```diff
@@ -103,7 +103,7 @@
         "comma-dangle": [ 2, "never" ],
         "comma-spacing": [ 2, { "before": false, "after": true } ],
         "comma-style": [ 2, "last" ],
-        "complexity": [ 2, 4 ],
+        "complexity": [ 2, 8 ],
```

Comment:
Please don’t change this rule globally. It is ok to make exception from this rule, but then please only for those functions where it is really necessary via `eslint-disable`-comment.

Reply:
This was erroring in plugins other than this one, so I set it to the highest value other plugins required. I can bring this back down and add `eslint-disable-line` to the other plugins if wanted

Reply:
I’ve fixed that problem in #149. So you just need to rebase from `master`.